src/displayapp/screens/Calendar.cpp

```diff
@@ -0,0 +1,95 @@
+/*  Copyright (C) 2024 thnikk, Boteium, JustScott
+
+    This file is part of InfiniTime.
+
+    InfiniTime is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published
+    by the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    InfiniTime is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#include "displayapp/screens/Calendar.h"
+#include "components/datetime/DateTimeController.h"
+
+using namespace Pinetime::Applications::Screens;
+
+Calendar::Calendar(Controllers::DateTime& dateTimeController) : dateTimeController {dateTimeController} {
+
+  // Create calendar object
+  calendar = lv_calendar_create(lv_scr_act(), NULL);
+  // Set size
+  lv_obj_set_size(calendar, LV_HOR_RES, LV_VER_RES);
+  // Set alignment
+  lv_obj_align(calendar, NULL, LV_ALIGN_IN_BOTTOM_MID, 0, -5);
+  // Disable clicks
+  lv_obj_set_click(calendar, false);
+
+  // Set style of today's date
+  lv_obj_set_style_local_text_color(calendar, LV_CALENDAR_PART_DATE, LV_STATE_FOCUSED, LV_COLOR_RED);
+
+  // Set style of inactive month's days
+  lv_obj_set_style_local_text_color(calendar, LV_CALENDAR_PART_DATE, LV_STATE_DISABLED, lv_color_hex(0x505050));
```

Comment:
Can we use a color from the InfiniTime theme here?

Reply:
I can replace the red with `Colors::deepOrange`, but `Colors::lightGray` doesn't look quite right. Would it be alright if I added something like `static constexpr lv_color_t gray = LV_COLOR_MAKE(0x50, 0x50, 0x50);` to the `Colors` namespace?

Reply:
Sounds good to me